backend/Model/RoomModel.php

```diff
@@ -49,7 +48,10 @@ class RoomModel extends AbstractModel
         'RoomID',
         'Rules',
         'MaxAmount',
-        'DateOfExchange'
+        'DateOfExchange',
+        'UserID',
+        'Rules',
```

Comment:
This one is already written 6 rows above

Reply:
It's not among fillable properties?
![image](https://user-images.githubusercontent.com/47412010/140475296-c1a67ba4-a98b-464d-ad06-7191919b6c14.png)